apps/meeting/server.js

```diff
@@ -60,24 +71,34 @@ const server = require(protocol).createServer(
         const name = query.name;
         const region = query.region || 'us-east-1';
 
+        let client = getClientForMeeting(meetingCache[title]);
         if (!meetingCache[title]) {
-          meetingCache[title] = await chime
-            .createMeeting({
-              ClientRequestToken: uuid(),
-              MediaRegion: region
-            })
-            .promise();
+          let request = {
+            ClientRequestToken: uuidv4(),
+            MediaRegion: region,
+            ExternalMeetingId: title.substring(0, 64),
+          };
+          if (query.ns_es === 'true') {
+            client = chimeRegional;
+            request.MeetingFeatures = {
+              Audio: {
+                // The EchoReduction parameter helps the user enable and use Amazon Echo Reduction.
```

Comment:
Could we make it two lines to reduce the line width?
And this comment seems to be too general, could we explain a bit more like how it helps?

Reply:
i see it is 100 characters so should be fine. this is also keeping the same from JS SDK. 
https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/server.js#L94